docs/excel/custom-functions-overview.md

```diff
@@ -353,10 +353,10 @@ function refreshTemperature(thermometerID){
 }
 ```
 
-## Co-Authoring
+## Coauthoring
 Excel Online and Excel for Windows with an Office 365 subscription allow you to co-author documents and this feature works with custom functions. If your workbook uses a custom function, your colleague will be prompted to load the custom function's add-in. Once you both have loaded the add-in, the custom function will share results through co-authoring.
```

Comment:
The body of this paragraph contains the term "co-authoring" which should be replaced with "coauthoring".

Suggest that you do a global (case-sensitive) search/replace throughout the doc set to:
- replace "co-authoring" with "coauthoring"
- replace "Co-authoring" with "Coauthoring"
- replace "Co-Authoring" with "Coauthoring"